### PR TITLE
Rename thread_priority to realtime_helpers header

### PR DESCRIPTION
**Pull request by saikishor**
Fixes: https://github.com/ros-controls/realtime_tools/issues/177

This is done to also have backward compatibility

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -39,6 +39,19 @@ target_include_directories(thread_priority PUBLIC
...
+)
+target_compile_features(realtime_helpers PUBLIC cxx_std_17)
+target_include_directories(realtime_helpers PUBLIC
+  $<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include/realtime_tools>
+)
+ament_target_dependencies(realtime_helpers PUBLIC ${THIS_PACKAGE_INCLUDE_DEPENDS})
+if(UNIX)
+  target_link_libraries(realtime_helpers PUBLIC cap)
+endif()
```
Comment (saikishor) [inline CMakeLists.txt:53]: @fmauch @christophfroehlich Do you think it is better to create a new library out of it or let's add it to the main library `realtime_tools` one

Comment (christophfroehlich) [reply CMakeLists.txt:53]: As you prefer..

Comment (saikishor) [reply CMakeLists.txt]: Done. Thanks

**Review by christophfroehlich — APPROVED** [review]

**Review by bmagyar — APPROVED** [review]
+1 for not adding a new library

**Pull request merged**